openflow13/match.go

```diff
@@ -141,6 +146,7 @@ func (m *MatchField) MarshalBinary() (data []byte, err error) {
 
 func (m *MatchField) UnmarshalBinary(data []byte) error {
 	var n uint16 = 0
+	var err error = nil
```

Comment:
Got it ; Yes, Go complier would init all declare  variable with 0 or nil (incase of reference type)